src/main/java/core/basesyntax/model/User.java

```diff
@@ -1,10 +1,25 @@
 package core.basesyntax.model;
 
 import java.util.List;
-
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+import org.hibernate.annotations.Cascade;
+import org.hibernate.annotations.CascadeType;
+
+@Entity
+@Table(name = "users")
 public class User {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
     private String username;
+    @OneToMany(fetch = FetchType.EAGER)
```

Comment:
Almost sure that isn't mistake. But we can use join rules here cause of bad naming of Hibernate.